packages/popover/src/types.ts

```diff
@@ -0,0 +1,123 @@
+import React from 'react';
+
+/**
+ * Options to determine the alignment of the popover relative to
+ * the other component
+ * @param Top will align content above other element
+ * @param Bottom will align content below other element
+ * @param Left will align content to the left of other element
+ * @param Right will align content to the right of other element
+ */
+const Align = {
+  Top: 'top',
+  Bottom: 'bottom',
+  Left: 'left',
+  Right: 'right',
+} as const;
+
+type Align = typeof Align[keyof typeof Align];
+
+const Justification = {
+  Top: 'top',
+  Bottom: 'bottom',
+  Left: 'left',
+  Right: 'right',
+  CenterVertical: 'center-vertical',
+  CenterHorizontal: 'center-horizontal',
+} as const;
+
+type Justification = typeof Justification[keyof typeof Justification];
+
+/**
+ * Options to determine the justification of the popover relative to
+ * the other component
+ * @param Start will justify content against the start of other element
+ * @param Middle will justify content against the middle of other element
+ * @param Bottom will justify content against the end of other element
+ */
+const Justify = {
+  Start: 'start',
+  Middle: 'middle',
+  End: 'end',
+} as const;
+
+type Justify = typeof Justify[keyof typeof Justify];
+
+export { Align, Justification, Justify };
```

Comment:
Oh sorry, I meant `export const Justify` etc.

Reply:
We're using Align, Justify and Justification as both a type and value so the export { Align } signature allows us to do that. Exported individually after the declarations though.